src/main/java/com/library/controller/mongo/BookStorageController.java

```diff
@@ -0,0 +1,45 @@
+package com.library.controller.mongo;
```

Comment:
BookController in "controller" package, BookStorageController in "controller.mongo" package.

Some discrimination or what?)

Reply:
discrimination is resolved